### PR TITLE
Implement app's commands

### PR DESCRIPTION
**Pull request by tamorim**
Depends on https://github.com/vtex/apps-client-node/pull/2

### CODE REVIEW

File: src/modules/apps.js [hunk 1]
```diff
@@ -7,40 +109,150 @@ export default {
     description: 'List your installed VTEX apps',
     handler: () => {
       log.debug('Starting to list apps')
-      log.info('You have no installed apps')
+      workspaceAppsClient().listDependencies(
+        getAccount(),
+        getDevWorkspace(getLogin()),
+        'storefront'
```
Comment (brenoc) [inline src/modules/apps.js]: Why filter by "storefront"?

File: src/modules/apps.js [hunk 2]
```diff
@@ -7,40 +109,150 @@ export default {
...
     description: 'Install the specified app',
-    handler: (app) => {
+    handler: (app, options) => {
       log.debug('Starting to install app', app)
-      log.info('Install app', app)
+      const appRegex = new RegExp(`^${vendorPattern}\.${namePattern}@${wildVersionPattern}$`)
+      if (!appRegex.test(app)) {
+        return log.error('Invalid app format, please use <vendor>.<name>@<version>')
+      }
+      const simulation = options.s || options.simulation
```
Comment (brenoc) [inline src/modules/apps.js]: What a "simulation" does?

File: src/modules/apps.js [hunk 3]
```diff
@@ -7,40 +109,150 @@ export default {
...
+        getAppManifest(root),
+        getVtexIgnore(root),
+      ])
+      .spread(({vendor, name, version}, ignore) => {
+        log.info('Watching app', `${vendor}.${name}@${version}`)
+        return all([
+          listFiles(root, ignore),
+          keepAppAlive({vendor, name, version}),
+        ])
+        .spread(localFiles => all([
```
Comment (brenoc) [inline src/modules/apps.js]: Where Render build step would come in here?

File: src/file.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    .on('ready', resolve)
+  })
+}
+
+export function sendSaveChanges (root, file, sendChanges) {
+  log.debug('Sending save changes', file)
+  return sendChanges(createChanges(root, { [file]: 'save' }))
+}
+
+export function sendRemoveChanges (root, file, sendChanges) {
```
Comment (brenoc) [inline src/file.js]: This file is incredibly big. I think we should split it. There's no way all this functions are cohesive.

**Comment by brenoc** [comment]
@firstdoit @tamorim what about my questions? lol

File: src/modules/apps.js (hunk 1 above)
Comment (tamorim) [inline src/modules/apps.js]: @brenoc Yeah, list only for storefront doesn't make sense.
BUT, a service is required. Should we make the service an argument?

File: src/modules/apps.js (hunk 2 above)
Comment (tamorim) [inline src/modules/apps.js]: @brenoc It's like a dry run flag, it won't install or uninstall the apps

File: src/modules/apps.js (hunk 3 above)
Comment (tamorim) [inline src/modules/apps.js]: @brenoc Probably after or in parallel with the `watch`

**Comment by firstdoit** [comment]
@brenoc I didn't knew the answers 😬

File: src/modules/apps.js (hunk 2 above)
Comment (firstdoit) [inline src/modules/apps.js]: why not call it dry run?

Comment (tamorim) [inline src/modules/apps.js]: @firstdoit We can rename it, i just named it after the API

**Pull request merged**